Jellyfin.Server/Program.cs

```diff
@@ -78,7 +96,7 @@ public static async Task Main(string[] args)
                 Shutdown();
             };
 
-            _logger.LogInformation("Jellyfin version: {Version}", version);
+            _logger.LogInformation("Jellyfin version: {Version}", Assembly.GetEntryAssembly().GetName().Version);
```

Comment:
I'm slightly wary of this change, plus it's not indicated in the PR. Please explain the reasoning

Reply:
It provides the same output to log, just avoids the var creations.

Reply:
I don't see you removing `version` variable anywhere, thus the question.